content/docs/use-cases/ci-cd-ml.md

```diff
@@ -0,0 +1,70 @@
+# Continuous Integration and Deployment for Machine Learning
+
+MLOps and DataOps apply DevOps methodologies to machine learning (ML) and data
+management. The fields are rapidly evolving beyond a mere set of best practices,
+and now encompass the entire product lifecycle management. A core component is
+the automation of testing and deployment: CI/CD (continuous integration and
+delivery).
+
+CI/CD is easy to set up with practically any Git repository. Some example
+scenarios are:
+
+- Agile development
+  - Automating quality assurance (QA) means teams can confidently deploy new
+    versions several times a day and even before the weekend without fear of
+    bugs/regressions
+- Easy testing
+  - When new data is pushed in a pull request (PR), validation tests
+    automatically run in the cloud (without each collaborator setting up
+    potentially inconsistent local testing environments)
+  - When a new model is pushed in a PR, reports (on performance against the
+    current deployed models) are automatically generated and posted as a PR
+    comment with graphs and tables
+- Continuous machine learning (CML): online training
+  - Schedule regular jobs pulling in data updates from public sources to refine
+    deployed models
+  - Provision bare metal in the cloud and run a brute force hyperparameter
+    search
+
+In ML, CI/CD can be used to automatically build, test and push data and models
+to production in the cloud with minimal effort. There are many components, such
+as:
+
+1. Data collection & storage
+2. Metadata management
+3. Validation & testing (including provisioning resources)
+4. Sharing & deployment
+5. Monitoring & feedback
+
+Quite often, CI/CD for ML is hard to set up and debug, and solutions unclear.
+DVC can alleviate most (in not all) of the
+[hidden technical debt in ML](https://papers.nips.cc/paper/2015/file/86df7dcfd896fcaf2674f757a2463eba-Paper.pdf).
```

Comment:
Great problem statement. Should probably be near the top of the doc.

Reply:
I think we can skip _(in not all)_, as it seems distracting.

Reply:
ah also typo... meant to be *(if not all)*

Reply:
> Great problem statement. Should probably be near the top of the doc.

Updated the PR description. The main priority is define MLOps (because for now it's still a relatively new term), then note problems, then propose solutions.

> I think we can skip _(in not all)_, as it seems distracting.

reworded.